{
	"vulinfoid": "JVNDB-2018-011633",
	"vulinfodata": {
		"title": "Kubernetes におけるエラー処理に関する脆弱性",
		"vulinfodescription": {
			"overview": "Kubernetes には、エラー処理に関する脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "Kubernetes",
					"productname": "Kubernetes",
					"cpe": {
						"text": "cpe:/a:kubernetes:kubernetes",
						"version": "2.2"
					},
					"versionnumber": [
						"1.10.11 未満",
						"1.11.5 未満",
						"1.12.3 未満"
					]
				},
				{
					"name": "レッドハット",
					"productname": "Red Hat OpenShift Container Platform",
					"cpe": {
						"text": "cpe:/a:redhat:openshift_container_platform",
						"version": "2.2"
					}
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.5",
					"vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "Critical",
						"type": "Base"
					},
					"base": "9.8",
					"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
				}
			],
			"impactitem": {
				"description": "情報を取得される、情報を改ざんされる、およびサービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "vendor",
					"name": "GitHub",
					"vulinfoid": "CVE-2018-1002105: proxy request handling in kube-apiserver can leave vulnerable TCP connections #71411",
					"url": "https://github.com/kubernetes/kubernetes/issues/71411"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3742",
					"url": "https://access.redhat.com/errata/RHSA-2018:3742"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3752",
					"url": "https://access.redhat.com/errata/RHSA-2018:3752"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3754",
					"url": "https://access.redhat.com/errata/RHSA-2018:3754"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3537",
					"url": "https://access.redhat.com/errata/RHSA-2018:3537"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3549",
					"url": "https://access.redhat.com/errata/RHSA-2018:3549"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3551",
					"url": "https://access.redhat.com/errata/RHSA-2018:3551"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3598",
					"url": "https://access.redhat.com/errata/RHSA-2018:3598"
				},
				{
					"type": "vendor",
					"name": "Red Hat Security Advisory",
					"vulinfoid": "RHSA-2018:3624",
					"url": "https://access.redhat.com/errata/RHSA-2018:3624"
				},
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2018-1002105",
					"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-1002105"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2018-1002105",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-1002105"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-388",
					"url": "https://cwe.mitre.org/data/definitions/388.html",
					"title": "エラー処理"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2019-01-18T11:21:47+09:00",
					"description": "[2019年01月18日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2019-01-18T11:21:47+09:00",
		"datelastupdated": "2019-01-18T11:21:47+09:00",
		"datepublic": "2018-11-26T00:00:00+09:00"
	}
}
